ccdaservice/package.json

```diff
@@ -12,6 +12,12 @@
         "lodash": "4.17.21",
         "net": "1.0.2",
         "uuid": "8.3.2",
-        "xmljson": "0.2.0"
+        "xmljson": "0.2.0",
```

Comment:
@sjpadgett , super cool stuff! and completely agree with keeping the services in the separate ccdaservice node_modules.
can the oe-cqm-service entry from main package.json now be removed?
at some point will need to create another official npm package for oe-schematron-service :) 

![2DV-2](https://user-images.githubusercontent.com/278968/168007402-a6cb8ef9-4fa2-40f2-af01-26d6019a6924.gif)

Reply:
yep will remove from core package. Have to register new npm packages because most times user won't have git installed.
Now I just need to remember how!:)